README.md

```diff
@@ -170,6 +170,25 @@ Dragonfly supports both **mirror** mode and HTTP **proxy** mode to boost the con
 
 `auth_through=false` means nydusd's authentication request will directly go to original registry rather than relayed by Dragonfly.
 
+### Hot update mirror configurations
+In addition to setting the registry mirror in nydusd's json configuration file, `nydus-snapshotter` also supports hot update mirror configurations. You can set the configuration directory in nudus-snapshotter's toml configuration file with `remote.mirrors_config`. The null string for `remote.mirrors_config` means disable it.
+```toml
+[remote.mirrors_config]
+dir = "/etc/nydus/certs.d"
+```
+Configuration file is compatible with containerd's configuration file in toml format.
+```toml
+server = "https://p2p-nydus.com"
+[host]
+  [host."http://127.0.0.1:65001"]
+    auth_through = true
```

Comment:
The suggested value of auth_through should be false

Reply:
Fixed.